### PR TITLE
tsp, example getbytes utf8

### PR DESCRIPTION
**Pull request by XiaofeiCao**
Change `getBytes()` to `getBytes(StandardCharset.UTF_8)` in examples.

### CODE REVIEW

**Comment by weidongxu-microsoft** [comment]
could you put 2 to another PR?

**Comment by XiaofeiCao** [comment]
> could you put 2 to another PR?

Sure.

File: javagen/src/main/java/com/azure/autorest/template/example/ModelExampleWriter.java [hunk 1]
```diff
@@ -146,6 +146,7 @@ public void accept(ExampleNode node, String getterCode) {
                 }
             } else if (node instanceof BinaryDataNode) {
                 this.imports.add(com.azure.core.util.BinaryData.class.getName());
+                this.imports.add(java.nio.charset.StandardCharsets.class.getName());
                 addEqualsAssertion(binaryDataNodeExpression((BinaryDataNode) node), getterCode);
```
Comment (weidongxu-microsoft) [inline javagen/src/main/java/com/azure/autorest/template/example/ModelExampleWriter.java:150]: Are you sure this is correct?

There is no `equals` in `BinaryData`?

Just assertNotNull?

Comment (XiaofeiCao) [reply javagen/src/main/java/com/azure/autorest/template/example/ModelExampleWriter.java:150]: I see... Just notNull then...

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**